src/ResourceManager/Sql/Commands.Sql/Instance Failover Group/Cmdlet/RemoveAzureSqlInstanceFailoverGroup.cs

```diff
@@ -0,0 +1,146 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.InstanceFailoverGroup.Model;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Management.Automation;
+using System;
+
+namespace Microsoft.Azure.Commands.Sql.InstanceFailoverGroup.Cmdlet
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureRmSqlDatabaseInstanceFailoverGroup",
+        SupportsShouldProcess = true)]
+    public class RemoveAzureSqlInstanceFailoverGroup : AzureSqlInstanceFailoverGroupCmdletBase
+    {
+        /// <summary>
+        /// Parameter set name for remove with an Input Object.
+        /// </summary>
+        protected const string RemoveIFGByInputObjectParameterSet =
+            "Remove a Instance Failover Group from AzureSqlInstanceFailoverGroupModel instance definition";
+        
+        /// <summary>
+        /// Parameter set name for remove with a resource ID.
+        /// </summary>
+        private const string RemoveIFGByResourceIdSet = "Remove a Instance Failover Group from Resource Id";
+
+        /// <summary>
+        /// Gets or sets the name of the resource group to use.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
```

Comment:
@jugeorge please remove the `ValueFromPipelineByPropertyName` attribute from the `ResourceGroupName`, `Location` and `Name` parameters, and make sure that you add them to a parameter set that is outside of the existing `InputObject` and `ResourceId` sets